### PR TITLE
Update #30

### PR DESCRIPTION
**Pull request by alialiwa2005**
Update to https://github.com/hackclub/winter/pull/30

### CODE REVIEW

File: alialiwa2005.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
 ---
 name: "@alialiwa2005"
-project: "Raspberry Web Services (RWS) | DIY Homemade AWS Cloud"
+project: "Libre Web Services (LWS) | DIY Homemade AWS Cloud" 
+(also known as "Raspberry Web Services (RWS)")
```
Comment (bellesea) [inline alialiwa2005.md:4]:
```suggestion
```

Comment (bellesea) [reply alialiwa2005.md:4]: we're using this section "metadata" for each project so we'd need to keep it standard with just the project name hehe, thank you!

File: alialiwa2005.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 ---
 
 # Project Name
```
Comment (bellesea) [inline alialiwa2005.md:7]:
```suggestion
# Libre Web Services (LWS) | DIY Homemade AWS Cloud (also known as "Raspberry Web Services (RWS)")
```

File: alialiwa2005.md [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
-Raspberry Web Services (RWS) | DIY Homemade AWS Cloud
+Libre Web Services (LWS) | DIY Homemade AWS Cloud
```
Comment (bellesea) [inline alialiwa2005.md:8]:
```suggestion
```

File: alialiwa2005.md [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
+(also known as "Raspberry Web Services (RWS)")
```
Comment (bellesea) [inline alialiwa2005.md:9]:
```suggestion
```

File: alialiwa2005.md [hunk 5]
```diff
@@ -109,13 +120,7 @@ Whichever path you choose to travel with this build, you can always HACK! :)
...
-| 4 Power Supplies                | https://bit.ly/3WwznmU                | $32     |
-| 6 Ethernet Cables               | https://bit.ly/3WtkAte                | $3.23   |
-| 4 MicroSD Cards                 | (Already Own)                         | $0      |
-| Customized Case                 | (Already Own Supplies)                | $0      |
-| Gigabit Ethernet Network Switch | (Already Own)                         | $0      |
-| Router for Project-Only Network | (Already Own a Spare Router)          | $0      |
-| Total Including NYC Tax (8.875%)|                                       | $277.89 |
+| Product                                 | Supplier/Link                         | Cost    |
+| --------------------------------------- | ------------------------------------- | ------- |
+| Libre Web Services                      | https://amzn.to/3vwx5ZI               | $277.37 |
```
Comment (bellesea) [inline alialiwa2005.md:125]: love that this is super convenient for ordering hehe but for records reasons, could you please break it out into individual components? also, is this total inclusive of estimated shipping and taxes?

thank you sm!

**Review by bellesea — CHANGES_REQUESTED** [review]
@alialiwa2005 this looks awesome! i'm wondering, are you following any sort of guide for this?

**Comment by bellesea** [comment]
> > @alialiwa2005 this looks awesome! i'm wondering, are you following any sort of guide for this?
> 
> Hi, @bellesea!
> 
> I'm using the same guides and building the same project as #30.
> 
> Only difference is I'm using Libre Computer boards as substitutes for the Raspberry Pis which are in a major shortage, and therefore, different power supply/cables (the Libre Computer uses microUSB unlike a Pi 4 which uses USB-C for power).
> 
> In addition to the references mentioned in #30, I'm also referencing Libre Computer's board-specific documentation.

@alialiwa2005 ahh cool, and which guides are those? could you please cite them as sources within your file? thank you sm!

**Review by bellesea — APPROVED** [review]
@alialiwa2005 thanks for updating it! approving it to move forward but please still commit the references for your project :) can't wait to see you work on this!

**Comment by alialiwa2005** [comment]
> @alialiwa2005 thanks for updating it! approving it to move forward but please still commit the references for your project :) can't wait to see you work on this!

Thank you! I've added the referenced sources below the budget section in my last commit.

**Review by hackliv — APPROVED** [review]

**Pull request merged**